framework/assets/yii.js

```diff
@@ -162,6 +162,10 @@ window.yii = (function ($) {
                 pjaxContainer,
                 pjaxOptions = {};
 
+            if (params !== undefined && params.hasOwnProperty('submit')) {
+                console.error("Property 'submit' of object #<HTMLFormElement> is not a function.");
```

Comment:
This error doesn't give you any clue on what to do to fix it thus it's not giving end user much.

Reply:
Any ideas?

Reply:
1. The error should say `Parameter name "submit" conflicts with a same named form property. Please use another name.`.
2. As it was described in original issue, the same is true for other elements:

> Forms and their child elements should not use input names or ids that conflict with properties of a form, such as submit, length, or method.

So it may make sense to expand condition.